docs/context/ref/BaseContext.md

```diff
@@ -8,7 +8,7 @@ sidebar_label: BaseContext
 
 ## Schema
 
-<https://github.com/finos/FDC3/tree/main/schemas/context/context.schema.json>
+https://github.com/finos/FDC3/tree/main/schemas/context/context.schema.json [https://github.com/finos/FDC3/tree/main/schemas/context/context.schema.json](https://github.com/finos/FDC3/tree/main/schemas/context/context.schema.json)
```

Comment:
This looks like a double link

Reply:
please review 

https://github.com/finos/FDC3/pull/1418